incubator/java-openliberty/image/project/validate.sh

```diff
@@ -65,39 +60,30 @@ the parent version in pom.xml, and test your changes.
   <parent>
```

Comment:
I think this is a less-than-ideal way of helping.   The message we just issued (lines 54-58) is going to be helpful on its own .   If we can't print something you can copy/paste let's not bother.    Cut out lines 60-65 I'd say.

Reply:
Done